ga/latest/kernel/Dockerfile.ubi9-minimal.ibmjava8

```diff
@@ -0,0 +1,193 @@
+# (C) Copyright IBM Corporation 2025.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# TODO
```

Comment:
Update this comment to match the commands we run internally so that external users can also build the base image by following the same.